.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,22 @@
+name: Build
+on: [push]
+
+jobs:
+  build:
+    name: Continous Integration
+    runs-on: ubuntu-latest
+    env:
+      SCRIPTS_PATH: .github/workflows/scripts/register_buildpack
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node Dependencies
+        shell: bash
+        run: |
+          pushd ${SCRIPTS_PATH}
+          npm install
+          popd
+      - name: Run tests
+        run: |
+          pushd ${SCRIPTS_PATH}
+          npm test
```

Comment:
Is there a way to get coverage reports? Maybe a separate issue but it would be good to know what level of confidence we should have on the mechanism that manages the registry.

Reply:
There's gotta be.  I can investigate.

Reply:
@jromero I added some code coverage during the CI/CI github action test run.   Here's where we're at:

<img width="649" alt="Screen Shot 2020-09-03 at 12 14 52 PM" src="https://user-images.githubusercontent.com/1181106/92151947-273cb000-eddf-11ea-9882-f09dc20d079d.png">


If there's anything else we're wanting to dive deeper into with regards to code coverage, perhaps we can do it as a follow-up PR.

Reply:
Awesome. That'll work. In the future, it would be nice to increase this an make it fail CI when coverage drops X percentage.

Reply:
Hmm... as I was thinking about how can we best track improvements to this automation if the issues we create here will try to run through this automation and be closed :D 

Just feedback, not something I think we should take action on in this PR.

Reply:
Exactly!!  I was thinking the same thing.  I did a quick google search to see if there was a way to filter Issues, but nothing came-up.  Surely there's got to be a way to do.